src/process/init.rs

```diff
@@ -213,6 +213,19 @@ pub fn container_init(args: ContainerInitArgs) -> Result<()> {
         }
     }
 
+    let mut do_chdir = !proc.cwd.is_empty();
+    // change directory to process.cwd if process.cwd is not empty
+    if do_chdir {
+        // This chdir must run before setting up the user.
+        // This may allow the user running youki to access directories
+        // that the container user cannot access.
+        match unistd::chdir(&*proc.cwd) {
+            Ok(_) => do_chdir = false,
+            Err(nix::Error::EPERM) => {}
+            Err(e) => return Err(anyhow::anyhow!("Failed to chdir: {}", e)),
+        };
+    }
```

Comment:
@guni1192  It's fine, but how about this code to remove a `mut`?
```suggestion
   let do_chdir = if proc.cwd.is_empty() {
        false
    } else {
        // This chdir must run before setting up the user.
        // This may allow the user running youki to access directories
        // that the container user cannot access.
        match unistd::chdir(&*proc.cwd) {
            Ok(_) => false,
            Err(nix::Error::EPERM) => true,
            Err(e) => bail!("Failed to chdir: {}", e),
    };
```

Reply:
That sounds better!